### ISSUE TITLE
[Bug]: `DismissableLayer.d.ts` file is missing when installing 1.4.4 via npm.

### ISSUE THREAD

**Issue by antonreshetov** [radix-vue/radix-vue#687]
### Environment

```bash
Developement/Production OS: Windows 10 19043.1110
Node version: 16.0.0
Package manager: pnpm@8.6.0
Radix Vue version: 1.0.0
Vue version: 3.0.0
Nuxt version: 3.0.0
Nuxt mode: universal
Nuxt target: server
CSS framework: tailwindcss@3.3.3
Client OS: Windows 10 19043.1110
Browser: Chrome 90.0.4430.212
```

### Link to minimal reproduction

none

### Steps to reproduce

`pnpm add radix-vue`

### Describe the bug

`DismissableLayer.d.ts` file is missing when installing them via npm.

### Expected behavior

`DismissableLayer.d.ts` should be

**Comment by sadeghbarati** [comment]
Seems working fine here without TypeScript error when I'm importing `Dialog` component

Tested with `dev` and `build` and `preview`

https://stackblitz.com/edit/vitejs-vite-pdhuie

---

But the `(.)` directory is strange in `radix-vue@1.4.4`

https://www.npmjs.com/package/radix-vue?activeTab=code

<details><summary>1.4.4</summary>
<p>

![image](https://github.com/radix-vue/radix-vue/assets/17789047/f5de98fd-92c4-4ba0-a037-0a1f07b26d20)

</p>
</details> 

<details><summary>1.4.3</summary>
<p>

![image](https://github.com/radix-vue/radix-vue/assets/17789047/e14a1f6e-6b1a-478a-88e4-8623d25cc882)


</p>
</details>

**Comment by antonreshetov** [comment]
@sadeghbarati I am facing an import type error when using the Context Menu from shadcn-vue, which in turn uses radix-vue components.

**Comment by zernonia** [comment]
Thanks @antonreshetov ! Yup the type was not generated because of some ts error that doesn't block the deployment. Let me look into that!

**Comment by zernonia** [comment]
Fixed in `1.4.5`!
